scio-test/src/test/scala/com/spotify/scio/testing/JobTestTest.scala

```diff
@@ -380,6 +390,16 @@ class JobTestTest extends PipelineSpec {
     }
   }
 
+  def testBigQueryJobReadingTheSameTableWthDifferentPredicateAndProjections: Unit =
+    JobTest[BigQueryJobReadingTheSameTableWthDifferentPredicateAndProjections.type]
+      .args("--input=table.in")
+      .input(BigQueryIO[TableRow]("table.in"), (1 to 3).map(newTableRow))
```

Comment:
add constructor for storage with predicate and projections

Reply:
👍 Yup that will make it look like:

```scala
BigQueryIO[TableRow]("table.in", selectedFields, rowRestriction), (1 to 3).map(newTableRow)
```

Reply:
done